plugins/obs-filters/CMakeLists.txt

```diff
@@ -1,111 +1,141 @@
 project(obs-filters)
 
-option(DISABLE_SPEEXDSP "Disable building of the SpeexDSP-based Noise Suppression filter" OFF)
+option(ENABLE_SPEEXDSP
+       "Enable building with SpeexDSP-based noise suppression filter" ON)
+option(ENABLE_RNNOISE "Enable building with RNNoise noise supression filter" ON)
 
-if(DISABLE_SPEEXDSP)
-	message(STATUS "SpeexDSP support disabled")
-	set(LIBSPEEXDSP_FOUND FALSE)
-else()
-	find_package(Libspeexdsp)
-	if(LIBSPEEXDSP_FOUND)
-		add_definitions(-DLIBSPEEXDSP_ENABLED)
-	endif()
+if(OS_WINDOWS)
+  option(
+    ENABLE_NVAVX
+    "Enable building with NVidia Audio FX filter (requires redistributable to be installed"
```

Comment:
At minimum, "Nvidia" should be "NVIDIA" here and throughout.

I actually think we should use "NVIDIA Noise Removal" or "NVIDIA Audio Effects SDK".  I don't know if we only implement NVIDIA Noise Removal or if we also implement NVIDIA Room Echo Removal.

The status message is missing the ending parenthesis.

Reply:
IIRC I copied the text verbatim from the current master branch - I can change the naming, but that means it's already "wrong" on master.

Reply:
I did notice that it was copied verbatim, and it is wrong on the master branch.  If you prefer, we can update it in a separate PR then.

Reply:
I used "NVIDIA Audio Effects SDK" for now.

Reply:
I'll open a separate PR for it.  It seems like something we can, and should, address on its own without tying it up in this.

Reply:
Of course - I adapted it already though, was easy enough. 😬